### PR TITLE
Mobile compatibility and design improvements

### PR DESCRIPTION
**Pull request by thelamer**
This PR is rough, I don't like going all in before collaboration. Consider everything in here just a coded version of a proposal for some tweaks to UI. These are some of the low hanging fruit issues I mentioned on discord, doesn't tackle anything serious.
* Reload Balances is tied to the function so it spams notification on initial load
* The BSC icon is just a placeholder using what is in source
* There are prettier errors I cannot get past for line breaks (carriage returns unknown)
* I see you removed refresh balances from non mobile version I think it is a very useful button
* I added @click stuff for link throughs but need a cursor modifier and something to let people know stuff is clickable.
* Footer needs a github icon (need more SVGs)

### CODE REVIEW

File: frontend/src/components/ChainInfo.vue [hunk 1]
```diff
@@ -35,15 +35,13 @@
...
 					</div>
-					<!--
 					<div class="col-3">
 						<q-btn @click="reloadBalances" color="primary" stack>
 							<q-icon name="img:ban-refresh.svg" size="3em" color="secondary" />
 							<div class="text-button">Refresh Balances</div>
 							<q-tooltip content-class="bg-positive">Refresh balances</q-tooltip>
 						</q-btn>
 					</div>
-					-->
```
Comment (wrap-that-potassium) [inline frontend/src/components/ChainInfo.vue:46]: The idea is actually to get rid of this button has balances should be updated by server-side events.
This was more confusing than anything else.

File: frontend/src/layouts/MainLayout.vue [hunk 2]
```diff
@@ -234,11 +221,18 @@ export default class MainLayout extends Vue {
 		await accounts.connectWalletProvider()
 	}
 
-	async disconnect() {
-		await accounts.disconnectWalletProvider()
-		await ban.setBanAccount('')
-		router.push('/')
+	openBsc(address: string) {
+		window.open(this.blockExplorerUrl + '/address/' + address, '_blank')
```
Comment (wrap-that-potassium) [inline frontend/src/layouts/MainLayout.vue:225]: There is a `openURL` utility in quasar. Could be rewritten as something like this:
```
openURL(`${this.blockExplorerUrl}/address/${address}`)
```
By default, this opens the URL in a new window.

File: frontend/src/store/modules/accounts.ts [hunk 3]
```diff
@@ -18,6 +18,7 @@ class AccountsModule extends VuexModule {
 	public activeBalance = 0
 	public chainId: string | null = null
 	public chainName: string | null = null
+	public blockExplorerUrl: string | null = null
```
Comment (wrap-that-potassium) [inline frontend/src/store/modules/accounts.ts:21]: `Networks.ts` in `frontend/src/utils` has definitions of networks.
We could reuse the first item in `blockExplorerUrls`.

**Comment by thelamer** [comment]
@wrap-that-potassium I Can't figure out how to render the BSC button on mobile, I do not know what logic is removing it, the monkey shows fine but not the BSC button.
I do not understand how to import constants from Networks.ts, can you explain that a bit more or commit changes directly to my branch? 
I also do not know what is breaking the "Withdrawal ban" button on mobile. 
```
ChainInfo.vue?1990:236 Withdrawal can't be done TypeError: Cannot read property 'validate' of undefined
    at VueComponent._callee3$ (ChainInfo.vue?1990:161)
    at tryCatch (runtime.js?750b:63)
    at Generator.invoke [as _invoke] (runtime.js?750b:293)
    at Generator.eval [as next] (runtime.js?750b:118)
    at asyncGeneratorStep (asyncToGenerator.js?1da1:3)
    at _next (asyncToGenerator.js?1da1:25)
    at eval (asyncToGenerator.js?1da1:32)
    at new Promise (<anonymous>)
    at VueComponent.eval (asyncToGenerator.js?1da1:21)
    at VueComponent.withdrawBAN (ChainInfo.vue?1990:161)
```

**Comment by wrap-that-potassium** [comment]
I will have a look at it tomorrow so that we can merge your work as this should bring back MetaMask mobile compatibility.
I'm still wondering how popups looks like on a mobile though :)

**Comment by thelamer** [comment]
You can just click on mobile mode in any web browser. https://developer.chrome.com/docs/devtools/device-mode/
I'm not debugging this from mobile for end to end compatibility, this just handles display stuff. Though as far as I understand it you don't need to change the metamask calls at all for them to work with mobile browser/wallet solutions.

**Comment by wrap-that-potassium** [comment]
I know. I used to work that way and when I tested this with my phone, other UX issues appeared :)
So I would test this again with my phone using MetaMask Android.
Would it be enough? Probably not, but a better release probably.

**Comment by thelamer** [comment]
Linking https://github.com/wBanano/wban-dApp/issues/5 and https://github.com/wBanano/wban-dApp/issues/4
There is a prompt now for BNB deposits just needs a number input added to select amount.
Used the Ban deposit dialogue as a wireframe.

**Comment by thelamer** [comment]
Linking https://github.com/wBanano/wban-dApp/issues/3

**Pull request merged**